Update 2016 tor blog post for v3 HS

AbstractBeliefs opened the pull request:
Searching for details on freenode's tor hidden service still brings up this blogpost relatively highly (not the highest, but still).

I think it's reasonable to edit the post to include the new v3 address - there is no reason for the v2 address to be used at this point, and we should work to ensure that online sources are consistent, up to date, and remain working. More details can be found in #453 .

I would like to somehow keep the old address, but strike it through. I don't think this is possible in a code block, which is considered pre-formatted. I'm open to suggestions.

swantzter commented:
i don't know if our markdown thing escapes html, but `<s></s>` should work I think?

AbstractBeliefs commented:
It does appear to escape HTML, `<pre><code>&lt;s&gt;abc&lt;/s&gt;</code></pre>` is generated.

swantzter commented:
Aw shoot.

AbstractBeliefs commented:
I thought about maybe using the appropriate unicode combining characters, if this is something that would pass review I'd take a punt at it.

swantzter commented:
I'd be worried that that could break copy/paste? Perhaps consider adding another section like the one that was removed in #453 saying "if you use an old version blah blah V2 address"?

AbstractBeliefs commented:
If it breaks copy paste, "perfect", from an opinionated pov. In Debian you have to go back to oldoldstable to get the last tor package that doesn't support v3, and in approximately a year it wont work at all. Using v2 should be a very deliberate decision, we've removed it elsewhere, mostly I just want to keep it here so it was clear what we've changed.

Example render in your browser: f̶r̶e̶e̶n̶o̶d̶e̶o̶k̶2̶g̶n̶c̶m̶y̶.̶o̶n̶i̶o̶n̶
Example render in my browser in the site:
![a rendered, struck out address](https://i.imgur.com/iM3CUsa.png "A rendered, struck out address")

swantzter commented:
Might depend on fonts, but that doesn't render right for me (on GitHub at least) Chrome 86, Win 10 Pro 2004
![](https://i.imgur.com/bE43uNi.png)

AbstractBeliefs commented:
Yes, this is why I was initially hesitant to use such an option. I guess this remains an open question.

swantzter commented on content/news/2016-09-18-resurrecting-tor-2.md:
```diff
@@ -38,3 +38,6 @@ clients—check their docs for instructions for now.
 connecting at least once over the Internet. We're investigating our options, and
 would like to provide a solution to this in the future.
+
+(Aug 2020 footnote: the old address was freenodeok2gncmy.onion . We have deprecated this following the release of Tor HS
```
```suggestion
(Aug 2020 footnote: the old address was `freenodeok2gncmy.onion`. We have deprecated this following the release of Tor HS
```
I think this would look better with styling

swantzter replied:
I leave this up to you, either change it, or ignore this comment and merge. Either is OK with me

AbstractBeliefs replied:
Done.
